lib/shopify-cli/shopifolk.rb

```diff
@@ -0,0 +1,85 @@
+require 'fileutils'
```

Comment:
Minor: We don't use `FileUtils` in the implementation of this, only in the test, so we can require it there instead.